astropy/modeling/functional_models.py

```diff
@@ -1370,3 +1463,116 @@ def custom_model_1d(func, func_fit_deriv=None):
             "All parameters must be keyword arguments")
 
     return custom_model(func, fit_deriv=func_fit_deriv)
+
+
+class Sersic2D(Fittable2DModel):
+    """
+    Two dimensional Sersic surface brightness profile.
+
+    Parameters
+    ----------
+    amplitude : float
+        Central surface brightness, within r_eff. 
+    r_eff : float
+        Effective (half-light) radius
+    n : float
+        Sersic Index.
+    x_0 : float, optional
+        x position of the center.
+    y_0 : float, optional
+        y position of the center. 
+    ellip : float, optional
+        Ellipticity.
+    theta : float, optional
+        Rotation angle in radians, counterclockwise from
+        the positive x-axis. 
+
+    
+    See Also
+    --------
+    Gaussian2D, Moffat2D, Lorentz2D
+
+    Notes
+    -----
+    Model formula:
+
+    .. math::
+
+        I(x,y) = I(r) = I_e\\exp\\left[-b_n\left(\\frac{r}{r_{e}}\\right)^{(1/n)}-1\\right]
+
+
+    The constant :math:`b_n` is defined such that :math:`r_e` contains half the total 
+    luminosity, and can be solved for numerically.  
+
+    .. math:: \\Gamma(2n) = 2\\gamma (b_n,2n)
+
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        from astropy.modeling.models import Sersic2D
+        import matplotlib.pyplot as plt
+
+        x,y = np.meshgrid(np.arange(100),np.arange(100))
+
+        mod = Sersic2D(amplitude = 1, r_eff = 25, n=4, x_0=50, y_0=50, ellip=.5,theta=-1)
+
+        img = mod(x,y)
+
+        plt.figure()
+        plt.imshow(np.log10(img),origin='lower',interpolation='nearest',cmap='binary_r')
```

Comment:
I think `astropy.visualization.LogStretch` an `ImageNormalize` should be used here instead of `np.log10(img)` (see section 'Matplotlib normalization' on http://astropy.readthedocs.org/en/latest/visualization/normalization.html#stretching for an example)